cmd/qat_plugin/README.md

```diff
@@ -119,6 +119,28 @@ $ kubectl apply -k https://github.com/intel/intel-device-plugins-for-kubernetes/
 > socket creation and kubelet registration. Furthermore, the deployments `securityContext` must
 > be configured with appropriate `runAsUser/runAsGroup`.
 
+#### Automatic Provisioning
+
+There's a sample [qat initcontainer](https://github.com/intel/intel-device-plugins-for-kubernetes/blob/main/build/docker/intel-qat-initcontainer.Dockerfile) that sets cfg_services and enables QAT VFs. To deploy, run as follows:
+
+```bash
+$ kubectl apply -k deployments/qat_plugin/overlays/qat_initcontainer/
+```
+
+The provisioning [script](https://github.com/intel/intel-device-plugins-for-kubernetes/blob/main/demo/qat-init.sh) and [template](https://github.com/intel/intel-device-plugins-for-kubernetes/blob/main/deployments/qat_plugin/overlays/qat_initcontainer/qat.conf) are available for customization. The script is mostly based on [qatlib](https://github.com/intel/qatlib)'s utility qat_init.sh, and the template for configuration is also following the configuration of qatlib.
+
+> **Note**: According to the changes of qatlib, these might be replaced or changed in the future.
+
+The provisioning config can be optionally stored in the ProvisioningConfig configMap which is then passed to initcontainer through the volume mount.
+
+There's also a possibility for a node specific congfiguration through passing a nodename via NODE_NAME into initcontainer's environment and passing a node specific profile via configMap volume mount.
```

Comment:
how to do it with the operator based deployment?

Reply:
@hj-johannes-lee this question was meant to point out that the controller code does not set `NODE_NAME` anywhere.

Reply:
fixed!